src/components/component.ts

```diff
@@ -274,9 +311,6 @@ export class Component {
    * Component, Table, and Group; render them immediately with .renderTo() instead.
    */
   public renderImmediately() {
-    if (this._clipPathEnabled) {
-      this._updateClipPath();
```

Comment:
No because the SVG spec says SVG elements are `overflow: hidden` by default.

Actually, Plottable 2.x set `overflow: visible` explicitly, and then restricted specific components from overflowing (the Plots, GuidelineLayer and SelectionBoxLayer). I've replicated that same behavior by replacing `_clipPathEnabled` with `_overflowHidden` which applies the `overflow: hidden` style onto the **content()** when true; the net behavior should be the same